src/components/Tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import Class from 'classnames';
+
+interface Props {
+  tabList: Tab[],
+  selectedTabId: string,
+  onTabSelected: (id: string) => void
+}
+
+export const Tabs: React.FC<Props> = React.memo(
+  ({ tabList, onTabSelected, selectedTabId }) => {
+    const findedTab = useMemo(
+      () => tabList.find(tab => tab.id === selectedTabId),
```

Comment:
You call this method two times in two different components, will be better to call it once
You can store in App state full selected tab and pass only it's id to this component
It should help to avoid calling find method in both of components